picard/ui/scripteditor.py

```diff
@@ -437,7 +437,7 @@ def update_example_listboxes(before_listbox, after_listbox, examples):
             before_listbox.addItem(before)
             after_listbox.addItem(after)
 
-    def display_examples(self, send_signal=True):
+    def display_examples(self):
         """Update the display of the before and after file naming examples.  Optionally emits an `update` signal.
 
         Args:
```

Comment:
@rdswift remove this part of the docstring too as `send_signal` is gone

Reply:
Done.  Thanks.